docs/process.md

```diff
@@ -0,0 +1,38 @@
+# Timeravel Software Development Process
+
+This file documents the overall process, procedures, rules, and conventions to be followed within the Timeravel project.
+
+
+## General Process Framework
+
+TBW:
+
+* SCRUM-"inspired"
+* dates / development cycles
+* meetings
+* planning
+
+
+## Task Management
+
+TBW:
+
+* template for writing issues
+* defining acceptance criteria / definition of done
+
+
+## Development Workflow
+
+TBW
+
+* Git Flow
+* Pair programming
+* Test-Driven Development
+
+
+## Coding Conventions
+
+TBW
+
+* Test classes
```

Comment:
Either change back Test to test or always stick to capitals in the beginning of lines, test is not an acronym.